client/test/server.go

```diff
@@ -317,17 +318,25 @@ func (cts *ClientTestServer) updateReq(w http.ResponseWriter, r *http.Request) {
 	log.Infof("got update request %v", r)
 	cts.Update.Called = true
 
+	// Enterprise client device provides post is not supported yet
+	if r.Method == "POST" {
+		w.WriteHeader(404)
+		return
+	}
```

Comment:
It would be nice to test that this fallback works.

Reply:
I would say this is actually tested in: https://github.com/mendersoftware/mender/blob/7e716aee59d0f7ff77008e3bb2f4f52906644e8c/app/mender_test.go#L303

Reply:
Doh! Of course it is, I was thinking of it backwards. :-P